assets/src/blocks/TableOfContents/TableOfContentsFrontend.js

```diff
@@ -0,0 +1,19 @@
+import {getTableOfContentsStyle} from './getTableOfContentsStyle';
+import {TableOfContentsItems} from './TableOfContentsItems';
+import {makeHierarchical} from './makeHierarchical';
+import {getHeadingsFromDom} from './getHeadingsFromDom';
+
+export const TableOfContentsFrontend = ({title, className, levels, table_of_contents_style}) => {
```

Comment:
Same comment as earlier, should remain submenu_style

Reply:
Change reverted.